robotpy_build/setup.py

```diff
@@ -2,7 +2,8 @@
 from os.path import abspath, exists, join
 from setuptools import find_packages, setup as _setup
 from setuptools_scm import get_version
-import toml
+from tomlkit import parse
+from tomlkit import dumps
```

Comment:
I would prefer not using from import for (de)serialisation things.

Reply:
Thanks for your replays first , What is your concern ? am I going in the right direction ? i would be happy to have answers

Reply:
So I can call the function as follow : 
1.import tomlkit 
2. tomlkit.loads()
3. tomlkit.parse()

good enough ?

Reply:
Yes, that seems like the standard way to use those functions.